opendrift/timer.py

```diff
@@ -6,9 +6,15 @@ class Timeable:
     Utility class for measuring total time spent in various steps in a class
     throughout program execution.
     """
+
     timers = OrderedDict()
     timing = OrderedDict()
```

Comment:
These can now be None.

Reply:
Getting a little tricky.
If these are None, `super().__init__()` must be added to both `Basemodel.__init__` and `Variables.__init__`
This is fine, but then there is problem with `netCDF_CF_unstructured`, which needs a timer before `super().__init__()` is called. But this cannot be called until after some data are collected....

Reply:
True, see https://github.com/OpenDrift/opendrift/pull/453 for a suggestion on how to do it with properties and avoiding using an __init__. Even with __init__ the original dict would be overwritten if used before __init__'ed.

Reply:
Yes, this is more elegant.

Reply:
Then we should remove this commit from this PR, but keep the other one (subset of times when importing).
Can that be done through GitHub?